### PR TITLE
Use mask based on bitwidth instead of u32 max constant

### PR DESCRIPTION
**Pull request by dark64**
Fixed https://github.com/Zokrates/ZoKrates/issues/785

CI failing because of wasm-bindgen dependencies (there is a fix already for this, hopefully lands soon)

### CODE REVIEW

File: zokrates_core/src/static_analysis/propagation.rs [hunk 1]
```diff
@@ -718,7 +720,9 @@ impl<'ast, T: Field> Folder<'ast, T> for Propagator<'ast, T> {
             UExpressionInner::Not(box e) => {
                 let e = self.fold_uint_expression(e).into_inner();
                 match e {
-                    UExpressionInner::Value(v) => UExpressionInner::Value((!v) & 0xffffffff),
+                    UExpressionInner::Value(v) => {
```
Comment (Schaeff) [inline zokrates_core/src/static_analysis/propagation.rs:723]: small format fix: these curly braces are not required

Comment (dark64) [reply zokrates_core/src/static_analysis/propagation.rs:723]: hmm interesting I don't remember adding those, it seems to have been added by `cargo fmt`

Comment (Schaeff) [reply zokrates_core/src/static_analysis/propagation.rs:723]: interesting, can you confirm that?

Comment (dark64) [reply zokrates_core/src/static_analysis/propagation.rs:723]: yep, just tested it

**Review by Schaeff — APPROVED** [review]

**Pull request merged**